Core/Src/porting/odroid_system.c

```diff
@@ -118,6 +118,8 @@ runtime_stats_t odroid_system_get_stats()
 void odroid_system_sleep(void)
 {
     odroid_settings_StartupFile_set(ACTIVE_FILE);
+    odroid_settings_MainMenuSelectedTab_set(0);
```

Comment:
Why is this done? Wouldn't it be a nice feature to remember this when going to sleep from the main menu as well?

Reply:
What I mean is that instead of setting 0, set the current tab and cursor :slightly_smiling_face:

Reply:
Oh yeah, that was a last minute fix for a minor issue that I found after I already submitted the pull request. I'll try to save the active tab instead 👍